platforms/avr/timer.c

```diff
@@ -78,6 +78,16 @@ inline void timer_clear(void) {
     }
 }
 
+/** \brief timer set
+ *
+ * FIXME: needs doc
+ */
+inline void timer_set(uint32_t time_ms) {
```

Comment:
```suggestion
void timer_set(uint32_t time_ms) {
```
`inline` in some C file won't actually work.